scripts/plop/templates/component/tests.hbs

```diff
@@ -1,8 +1,4 @@
 import { expect, fixture, html, waitUntil } from '@open-wc/testing';
-// import sinon from 'sinon';
-
-import '../../../dist/shoelace.js';
```

Comment:
Two different reasons.

* sinon wasn't used in most test files, so I figured it's easier to add to the few times where it is used rather than having to delete it from all of the files where it isn't.
* And the second import is now global! Very excited about that one, it is in the `web-test-runner.config.js` HTML so it can be loaded "globally" rather than needing to be loaded in every test file. So this one is covered elsewhere.

Reply:
Good point. I did notice this, but it was the very last file in the list for review and I was a bit scared to go searching for this to update it out of fear I'd lose the rest of my comments. 😅